Dockerfile

```diff
@@ -44,6 +44,9 @@ WORKDIR $GOPATH/src/github.com/oysterprotocol/brokernode
 
 RUN go get -u github.com/gobuffalo/buffalo/buffalo
 
+# # Prometheus client:
+RUN go get -d -v github.com/prometheus/client_golang/prometheus
```

Comment:
Isn't this fetched from ` RUN go get -t -d -v ./...` below?

Reply:
I dont know what you mean, its needy to have it here to run prometheus.. how would you like to have it?

Reply:
I will take a look what lines below do exactly nad will try to remove